docs/debian12-install-guide.md

```diff
@@ -88,11 +88,14 @@ sudo mysql_secure_installation
 sudo mysql -u root -p
 ```
 - Enter the root password you just set up.
-- This root user has remote access disabled, so we will create a new "acore" user for the SQL database.
+- This root user has remote access disabled, so we will create a new "acore" user for the SQL database and the tables for AzerothCore.
 ```bash
 DROP USER IF EXISTS 'acore'@'localhost';
 CREATE USER 'acore'@'%' IDENTIFIED BY 'SQLPASSWORD';
 GRANT ALL PRIVILEGES ON * . * TO 'acore'@'%';
+CREATE DATABASE `acore_world` DEFAULT CHARACTER SET UTF8MB4 COLLATE utf8mb4_unicode_ci;
+CREATE DATABASE `acore_characters` DEFAULT CHARACTER SET UTF8MB4 COLLATE utf8mb4_unicode_ci;
+CREATE DATABASE `acore_auth` DEFAULT CHARACTER SET UTF8MB4 COLLATE utf8mb4_unicode_ci;
```

Comment:
Why not use the default sql setup from the git script here?

Reply:
Trying to keep things as short as possible. It has the same result but omits the `max_x_per_hour 0` and second set of `grant all privileges on 'acore_`  because they're redundant.
I will change the `@'%'` to default `@'localhost'` though and put a link to the ssh tunnelling guide.

Reply:
Yeah that makes sense, right now the delete user and create user queries do not match on the guide.